keras/callbacks/model_checkpoint_test.py

```diff
@@ -448,6 +448,12 @@ def get_model():
     )
     @pytest.mark.requires_trainable_backend
     def test_model_checkpoint_loading(self):
+        # TODO This test sometimes FAILED for Torch and Jax
```

Comment:
We've investigated this a bit but couldn't identify the cause.

Reply:
I will do this task tomorrow.
I will make small unit tests until I find the bug. I enjoy that :)